### PR TITLE
refactor: remove admin role and clarify maintainer

### PR DESCRIPTION
**Pull request by tiagolobocastro**
Removes the Admin Roles as Maintainers are essentially Admins. A maintainer is only expected to make substancial contributions, but not necessarily in a full-time position.
Add missing special maintainer from table.

### CODE REVIEW

File: GOVERNANCE.md [hunk 1]
```diff
@@ -4,48 +4,47 @@
...
 
 ---
 <BR>
 
-The **OpenEBS project** is managed on a daily basis by a senior governance & leadership team, that consists of the following OpenEBS ```Admins``` and ```Maintainers.``` Currently the team is...
+The **OpenEBS project** is managed on a daily basis by a senior governance & leadership team, that currently consists of the following OpenEBS ```Maintainers```:
 > |   |   |   |    |
 > | :--- | :--- | :--- | :--- |
-> | [Vishnu Attur](https://www.linkedin.com/in/vishnu-attur-5309a333/ "Bengaluru, Karnataka, India (GMT+5:30) Timezone")| :octocat: <kbd>**[@avishnu](https://github.com/avishnu "Vishnu Govind Attur")**</kbd> | ![](/images/flags/de_je/in.png) | <kbd>**Admin**</kbd>, ```Maintainer``` |
+> | [Vishnu Attur](https://www.linkedin.com/in/vishnu-attur-5309a333/ "Bengaluru, Karnataka, India (GMT+5:30) Timezone")| :octocat: <kbd>**[@avishnu](https://github.com/avishnu "Vishnu Govind Attur")**</kbd> | ![](/images/flags/de_je/in.png) | ```Maintainer``` |
```
Comment (tiagolobocastro) [inline GOVERNANCE.md:15]: Ah looks like this is also repeated on the README.md
Must we keep this on both?

Comment (avishnu) [reply GOVERNANCE.md:15]: IMO let it be.

Comment (avishnu) [reply GOVERNANCE.md:15]: Do we need to fix the admin bit in the readme as well?

**Review by niladrih — APPROVED** [review]
I don't have much context on this change. However, looks like it simplifies things, and doesn't take away from any key governance function. LGTM.

File: GOVERNANCE.md [hunk 2]
```diff
@@ -4,48 +4,47 @@
...
 
 ## Key references:
-Please familiarize yourself with our [Project Vision](./VISION.md) and [Contributing Guidelines](./CONTRIBUTING.md). Together with this Governance document, these provide useful info on what and who we are, and how we operate. 
+Please familiarize yourself with our [Project Vision](./VISION.md) and [Contributing Guidelines](./CONTRIBUTING.md). Together with this Governance document, these provide useful info on what and who we are, and how we operate.
 
 ---
 <BR>
 
-The **OpenEBS project** is managed on a daily basis by a senior governance & leadership team, that consists of the following OpenEBS ```Admins``` and ```Maintainers.``` Currently the team is...
+The **OpenEBS project** is managed on a daily basis by a senior governance & leadership team, that currently consists of the following OpenEBS ```Maintainers```:
```
Comment (Abhinandan-Purkait) [inline GOVERNANCE.md]: ```Maintainers```  and  ```Special Maintainers``` ?

**Review by Abhinandan-Purkait — APPROVED** [review]

**Review by avishnu — APPROVED** [review]
We need a change request to remove the 'Admin' from https://github.com/openebs/openebs?tab=readme-ov-file#get-in-touch-with-our-team as well.

**Comment by orville-wright** [comment]
I'm not following this change. 

The Admin role is a CNCF function that CNCF gives to the maintainers so that they can manage the project on behalf of CNCF so that CNCF doesn't have to spend their time managing projects (it's a kind of special delegated CNCF leadership status). 
Therefore Admins and maintainers are the superusers of the project and are kind of similar, but not 100% the same.

It's not clear how ADMIN and MAINTAINER roles will be differentiated in your proposal and how the 2 roles are managed?

Also, I'd like to see some examples of other CNCF projects that have done this, to feel comfortable that this is a generally accepted common practice within CNCF projects that leads to healthy governance practices. 

Finally, why are we obsessing about this kind of structural changes when there's much bigger &
more important t problems to solve, like: Promotion to Sandbox.

File: GOVERNANCE.md (hunk 2 above)
Comment (tiagolobocastro) [reply GOVERNANCE.md]: Done

File: GOVERNANCE.md (hunk 1 above)
Comment (tiagolobocastro) [reply GOVERNANCE.md:15]: Done

**Comment by tiagolobocastro** [comment]
> It's not clear how ADMIN and MAINTAINER roles will be differentiated in your proposal and how the 2 roles are managed?

Admin role is removed and becomes part of Maintainer role:
```
Maintainers also perform the admin non-code-related Github tasks such as updating permissions, configuring integrations and other administrative tasks.
```

> Also, I'd like to see some examples of other CNCF projects that have done this, to feel comfortable that this is a generally accepted common practice within CNCF projects that leads to healthy governance practices.

This brings it more in line with other CNCF projects

**Comment by orville-wright** [comment]
@tiagolobocastro cab you provide a few examples of the CNCF projects you allude to that also operate this way?

Also, please confirm if I understand this correctly - this change now gives all Maintainers the power of CNCF project admins?

**Comment by tiagolobocastro** [comment]
> @tiagolobocastro cab you provide a few examples of the CNCF projects you allude to that also operate this way?

helm, jaeger, etcd, etc...
I think it'd harder to find one which doesn't operate this way.

> Also, please confirm if I understand this correctly - this change now gives all Maintainers the power of CNCF project admins?

Yes, although that's already kinda true today. CNCF can give admin rights to a maintainer on request. So this effectively means a maintainer is an admin, there's no need to separate this, and as you can see from various other CNCF projects, they usually don't.

**Review by orville-wright — APPROVED** [review]

**Comment by orville-wright** [comment]
Thanks for the supporting info @tiagolobocastro  - its seems good that we align with CNCF best practices as much a we can.

**Pull request merged**